app/src/main/scala/org/alephium/explorer/persistence/queries/TransactionQueries.scala

```diff
@@ -313,6 +313,17 @@ object TransactionQueries extends StrictLogging {
     } yield txs
   }
 
+  def streamTxIds(address: Address, from: TimeStamp, to: TimeStamp): StreamAction[TransactionId] = {
+    sql"""
+      SELECT tx_hash
+      FROM transaction_per_addresses
+      WHERE address = $address
+      AND block_timestamp >= $from
+      AND block_timestamp <= $to
```

Comment:
That's also always a recurrent question in my head ^^ pros and cons always fights. Any opinion @mvaivre ?

Reply:
I would personally include to :) 

When you want to download you year worth of TXs, you want to list from `01.01.22` to `31.12.22`. IMO that's the "natural" way to handle it. Would be weird to set the to at `01.01.23`, but that's just my opinion :))

Reply:
However, I believe on UI, it's more intuitive to include `to`. Like zero-based indexing for developers, and one-based indexing for users.

Reply:
@mvaivre for your example, if we include `to` it means you will need to send from `01.01.22-00:00:000Z` to`31.12.22-23:59:999Z`.
Thinking a bit out loud, maybe on explorer-backend side it would be better to have the `to` exclusive: `to : 01.01.23-00:000Z` so it's easier for people manually calling the endpoint and anyway on the front-end side you can display whatever you want in the UI, you won't precise the hour for example, it will be day by day I guess?

Reply:
Yeah ok! Sounds good, let's exclude `to` inn the backend, and include it on the UI 👌